utils.js

```diff
@@ -46,16 +47,23 @@ module.exports = {
    * Find and extract headers
    * @param {object} reg
    */
-  getHeaderOptions: function(req){
-    //I have to remove the User-Agent header ever since superagent 1.7.0
-    if(req && req._header){
-      return this.pruneObj(req._header, ['User-Agent', 'user-agent']);
+  getHeaderOptions: function(_req){
+    // I have to remove the User-Agent header ever since superagent 1.7.0
+    // The cache-control header must also be removed.
+    // Clone the request first, as we don't want to remove the headers from the original one, do we?
+    const req = JSON.parse(JSON.stringify(_req));
+    const headersToPrune = ['User-Agent', 'user-agent', 'Cache-Control', 'cache-control'];
```

Comment:
Is this not normalized before? `UsEr-AgEnt` would technically be viable too.

This needs to be checked with `.toLowerCase` or something then